Tests/LibAudio/TestPlaybackStream.cpp

```diff
@@ -8,17 +8,30 @@
 #include <AK/MemoryStream.h>
 #include <AK/WeakPtr.h>
 #include <LibAudio/PlaybackStream.h>
+#include <LibCore/EventLoop.h>
 #include <LibTest/TestSuite.h>
 #include <unistd.h>
 
 #if defined(HAVE_PULSEAUDIO)
 #    include <LibAudio/PulseAudioWrappers.h>
 #endif
 
-TEST_CASE(create_and_destroy_playback_stream)
+// FIXME: CI doesn't run an AudioServer currently. Creating one in /etc/SystemServer.ini does not
+//        allow this test to pass since CI runs in a Shell that will setsid() if it finds that the
+//        current session ID is 0, and AudioServer's socket address depends on the current sid.
+//        If we can fix that, this test can run on CI.
```

Comment:
seems worth an actual issue. I know we aren't the best about actually dispositioning and prioritizing those, but this seems like a pretty beefy yak

Reply:
Fair, lemme copy this into a new issue. I'll leave the fixme, though, in case someone notices that and wants to look into it.

Reply:
https://github.com/SerenityOS/serenity/issues/20538